label-issue.js

```
@@ -0,0 +1,104 @@
+ */
+const { getIssueContent, applyLabels } = require('./lib/github-wrapper');
+const { callModel } = require('./lib/github-model');
+const fs = require('fs');
+const path = require('path');
+const { Command } = require('commander');
+
+// Read the prompt template
+// Remove the synchronous file read at the top of the file.
+// Inside the async main function (at its start), add:
+const promptTemplate = await fs.promises.readFile(path.join(__dirname, 'prompt-template.txt'), 'utf8');
+let promptTemplate;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'promptTemplate'. Consider to delete it or rename it.

'promptTemplate' is defined here:


(lint/suspicious/noRedeclare)